home/tests/test_whatsapp.py

```diff
@@ -268,6 +279,30 @@ def test_create_standalone_whatsapp_template(self) -> None:
         assert request.headers["Authorization"] == "Bearer fake-access-token"
         assert json.loads(request.body) == data
 
+    @responses.activate
+    def test_standalone_template_name(self) -> None:
+        """
+        Creating a WhatsApp template results in a single HTTP call to the
+        WhatsApp API containing the template data.
+        """
+        data = {
+            "category": "UTILITY",
+            "name": "Test template 1",
+            "language": "en_US",
+            "components": [{"type": "BODY", "text": "Test Body"}],
+        }
+        url = "http://whatsapp/graph/v14.0/27121231234/message_templates"
+        responses.add(responses.POST, url, json={})
+
+        locale = Locale.objects.get(language_code="en")
+        create_standalone_whatsapp_template(
+            "Test template 1", "Test Body", "UTILITY", locale=locale
+        )
```

Comment:
Should we maybe assert on the response from this function? So that we can test our success message?